MdePkg/Include/Register/Intel/StmApi.h

```diff
@@ -15,11 +15,14 @@
 #include <Register/Intel/StmStatusCode.h>
 #include <Register/Intel/StmResourceDescriptor.h>
 #include <Register/Intel/ArchitecturalMsr.h>
+#include <Base.h>
 
 #pragma pack (1)
 
 // MU_CHANGE - Added SMM_REV_ID definition, according to STM spec
 #define STM_SMM_REV_ID  0x80010100
+// MU_CHANGE - Added signature for CPU_INFORMATION_HEADER
+#define CPU_INFORMATION_HEADER_SIGNATURE  SIGNATURE_32('C', 'p', 'u', 'I')

```

Comment:
Because this is C, it'd be better to prefix the macro to scope it.

```suggestion
#define STM_CPU_INFORMATION_HEADER_SIGNATURE  SIGNATURE_32('C', 'p', 'u', 'I')
```